scripts/stateful/generate_upgrade_matrix_v15.py

```diff
@@ -4,7 +4,7 @@
 import sys
 import re
 
-SKIP_STARTING_VERSIONS = ['v14.0.0','v15.0.0-rc0','v15.0.0-rc1','v15.0.0-rc2','v15.0.0-rc3','v15.0.0']
+SKIP_STARTING_VERSIONS = ['v14.0.0','v14.1.0','v15.0.0-rc0','v15.0.0-rc1','v15.0.0-rc2','v15.0.0-rc3','v15.0.0']
 SKIP_TARGET_VERSIONS = ['v15.0.0-rc0','v15.0.0-rc1','v15.0.0-rc2']
```

Comment:
We can go straight to v15.0.0 now:
```suggestion
SKIP_TARGET_VERSIONS = ['v15.0.0-rc0','v15.0.0-rc1','v15.0.0-rc2','v15.0.0-rc3']
```